<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-E8IIGOH" name="Test" tests="2" failures="1" timestamp="9 Feb 2018 14:20:17 GMT" time="55.942" errors="0">
  <testcase name="@AfterMethod getResult" time="0.0" classname="testCases.DeletingUser">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at utility.GetScreenshot.capture(GetScreenshot.java:16)
	at utility.BaseExtentTest.getResult(BaseExtentTest.java:44)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="createUser" time="21.208" classname="testCases.CreateUser"/>
  <testcase name="deleteUser" time="5.016" classname="testCases.DeletingUser">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
	at testCases.DeletingUser.deleteUser(DeletingUser.java:71)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- deleteUser -->
</testsuite> <!-- Test -->
